sdk/eventhub/event-hubs/package.json

```diff
@@ -40,15 +40,15 @@
     "build:samples": "cd samples && tsc -p .",
     "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
     "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build:test": "npm run build",
+    "build:test": "npm run build:test:node",
```

Comment:
This used to have `&& npm run build:test:browser` as well.
@ShivangiReja saw build errors when this was included.
May be your current fix fixed this as well?

Reply:
I only updated the config for node.js, not for browser builds. I think we would actually need to bundle `@azure/identity` for browser tests so we can get it into a browser runtime.

Reply:
got it

@ShivangiReja, please log another issue for the build problems you saw when browser was being built after adding identity tests